planner/planner_APIs.go

```diff
@@ -440,6 +470,67 @@ func (planner *MyPlanner) getPlanningApi(ctx *gin.Context) {
 	utils.LogErrorWithLevel(planner.ResultHTMLTemplate.Execute(ctx.Writer, planningResp), utils.LogError)
 }
 
+func (planner *MyPlanner) getPlanDetails(c *gin.Context) {
+
+	var id string = c.Param("id")
+	iowrappers.Logger.Debugf("GET Route /plans/%s", id)
+
+	var planRecordRedisKey = strings.Join([]string{iowrappers.TravelPlanRedisCacheKeyPrefix, id}, ":")
+	cachePlanSolution, cacheErr := planner.RedisClient.GetCachePlan(c, planRecordRedisKey)
+	if cacheErr != nil {
```

Comment:
The error source is not limited to `KeyNotFound` error

Reply:
Thanks, I have updated debug msg for this error.